app/models/location.rb

```diff
@@ -21,7 +21,12 @@ def to_s(letter_style: false)
 
   # rubocop:disable Style/RedundantSelf
   def ==(other)
-    other.class.eql?(self.class) && self.to_s.eql?(other.to_s)
+    other.class.eql?(self.class) &&
```

Comment:
Why did you make this change, especially since Rubocop has issues with the amount of branching conditions?

Reply:
Oh I did it because I thought it would be more correct to compare values as opposed to the stringified version of the class. But I'm happy to revert it back!

Reply:
Yes, could you revert it back please?  In general, I think equivalent string values will be enough in this case.

Reply:
Done!